e2e-version/README.md

```diff
@@ -0,0 +1,97 @@
+# E2E Grafana version resolver
+
+This Action resolves what versions of Grafana to use when E2E testing a Grafana plugin in a Github Action.
+
+## Features
+
+The action supports two modes.
+
+**plugin-grafana-dependency (default)**
+The will return all minor versions of Grafana that have been released since the version that was specified as grafanaDependency in the plugin.json. This requires the plugin.json file to be placed in the `<root>/src` directory.
```

Comment:
I'm also thinking the same as @academo regarding over time this producing a large number of jobs. If I create a plugin that is compatible with Grafana 8 and I set the `grafanaDependency: >=8.0.0` what does the matrix look like?

Reply:
Today the output would be:
```json
["10.2.2","10.1.5","10.0.9","9.5.14","9.4.17","9.3.16","9.2.20","9.1.8","9.0.8","8.5.27","8.4.11","8.3.11","8.2.7","8.1.8","8.0.7"]
```

So yes, a lot of jobs. I guess your main concern here is computational resources? 

If we look at this from a plugin author's perspective, I think it makes sense to run E2E tests against all versions of Grafana that your plugin is supposed to have compatibility with (according to grafanaDependency in plugin.json). That being said, I _think_ many internal plugins haven't touched the grafanaDependency prop in plugin.json since they the project started, so in most cases it's probably no longer accurate. 

In case the actions is used with the `version-support-policy` mode,  the output today would be
```json
["10.2.2","10.1.5","10.0.9","9.5.14"]
```
Maybe this should be the default resolver instead? 
cc @academo

Reply:
Yeah that's a good idea. I'll try to come up with a proposal.